subjects/multorsum/README.md

```diff
@@ -0,0 +1,35 @@
+## multorsum
+
+### Instructions
+
+Write a function that receives a slice of int and and int representing the initial value. If the int is odd it should multiply the accumulated value. If it is even, it should be added instead. Return the result. If the slice is empty return 0.
```

Comment:
> If the slice is empty return 0.

The result of your solution `MultOrSum([]int{}, -100)` ==> `-100` not `0`

Reply:
Good catch thanks. I'll fix it.